lib/net.js

```diff
@@ -57,10 +57,10 @@ const exceptionWithHostPort = util._exceptionWithHostPort;
 
 function noop() {}
 
-function createHandle(fd) {
+function createHandle(fd, is_server) {
   const type = TTYWrap.guessHandleType(fd);
-  if (type === 'PIPE') return new Pipe();
-  if (type === 'TCP') return new TCP();
+  if (type === 'PIPE') return new Pipe(is_server);
+  if (type === 'TCP') return new TCP(is_server);
```

Comment:
Rather than using a boolean, I'd much prefer an enum type value, similar to what we have with `Http2Session`, which either has type `NGHTTP2_SESSION_SERVER` (value `0`) or `NGHTTP2_SESSION_CLIENT` (value `1`).

Reply:
Sure, I will take a look :)